dcos-labs/ui-kit
# feat: use shared tslint-config

**How to test**:
- `npm i`
- Go to `node_modules/tslint-config`
- Change a rule e.g `no-consecutive-blank-lines` to `false`
- Go to badge component and then just add consecutive blank lines

run `npm run lint` and you should see failing with `no-consecutive-blank-lines` if `true`

**Closes DCOS-21940**
